Documentation/building/code-coverage.md

```diff
@@ -70,12 +70,13 @@ The following steps can be used manually to produce a coverage report, but a cus
 1. Build the local test project (`msbuild /T:Build`)
 3. Build coreclr locally in Debug or Release (`build.cmd all Debug skiptests`)
 2. Navigate to the built test directory in the corefx bin (e.g. `bin/tests/AnyOS.AnyCPU.Debug/System.Runtime/netcoreapp1.0` for `System.Runtime`
-4. Delete `coreclr.dll`, `mscorlib.dll` and `mscorlib.ni.dll` from that directory
-5. Copy all files in the coreclr bin directory to the test directory
-6. Run an OpenCover command with `xunit.console.netcore.exe`. For example:
+4. Delete `coreclr.dll`, `mscorlib.dll`, `mscorlib.ni.dll`, `System.Private.CoreLib.dll` and `System.Private.CoreLib.ni.dll` from that directory
```

Comment:
Hmm. It's strange to me that an overwrite would be treated as a modification rather than as a deletion+creation. But I trust Matt's explanation. Thanks.